src/Polly.Core/CircuitBreaker/BrokenCircuitException.cs

```diff
@@ -22,6 +22,14 @@ public BrokenCircuitException()
     {
     }
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="BrokenCircuitException"/> class.
+    /// </summary>
+    /// <param name="retryAfter">The time during which the circuit will be broken.</param>
```

Comment:
While true, I think it's describing a related concept rather than the literal meaning of the value. Something like the below might be better:

```suggestion
    /// <param name="retryAfter">The period after which the circuit will close.</param>
```

Whatever the change is to, it should be applied to all the constructors not just this one.